### PR TITLE
f-registration@v0.60.0 - Revised API mocking within stories to work with new module

### PR DESCRIPTION
**Pull request by tommcclean**
This PR revises the API mocking in the registration story file so that mocks can still be created and work.

**storybook**
v0.41.0
------------------------------
*August 10, 2021*

### Added
- Add the f-http mockFactory to process within Storybook to support story API mocks

<hr>

**f-http**
v0.8.0
------------------------------
*August 09, 2021*

### Changed
- Converted from vue-cli-service to vite

v0.7.0
------------------------------
*August 05, 2021*

### Added
- Mock Factory: Now supports passing through of requests

### Changed
- Mock Factory: Setup Mock Response does not require request data to be provided

### Fixed
- Correctly point the main entry point at the dist folder

<hr>

**f-registration**
v0.60.0
------------------------------
*August 05, 2021*

### Changed
- Update the story file API mocking to work with f-http

<hr>

**f-statistics**
v0.2.0
------------------------------
*August 09, 2021*

- ### Fixed
- Change build process to Vite

v0.1.1
------------------------------
*June 16, 2021*

- ### Fixed
- Correctly point the main entry point at the dist folder

### CODE REVIEW

File: packages/services/f-statistics/package.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
   "name": "@justeat/f-statistics",
   "description": "Javascript client for transporting statistics",
-  "version": "0.1.0",
-  "main": "src/index.js",
-  "maxBundleSize": "100kB",
```
Comment (dandel10n) [inline packages/services/f-statistics/package.json:6]: I think `maxBundleSize` is used by bundlewatch so need to stay

File: packages/storybook/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 v0.40.0
 ------------------------------
+<<<<<<< HEAD
```
Comment (dandel10n) [inline packages/storybook/CHANGELOG.md]: Oops

Comment (tommcclean) [reply packages/storybook/CHANGELOG.md]: Cheers :D I need to revisit the changelogs shortly; got one more issue to resolve before its ready

File: packages/services/f-statistics/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,21 @@
...
 
+v0.2.0
+------------------------------
+*August 09, 2021*
+
+- ### Fixed
+- Change build process to Vite
+
+
+v0.1.1
```
Comment (dandel10n) [inline packages/services/f-statistics/CHANGELOG.md:14]: Can this be part of v0.2.0 as the v0.1.1 won't be published separately?

Comment (tommcclean) [reply packages/services/f-statistics/CHANGELOG.md:14]: v0.1.1 was published; I needed to publish it prior to the PR to try and establish the issue

**Review by dandel10n — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

File: packages/services/f-http/package.json [hunk 4]
```diff
@@ -1,11 +1,12 @@
 {
   "name": "@justeat/f-http",
   "description": "Javascript HTTP client for interacting with restful services",
-  "version": "0.6.1",
-  "main": "src/index.js",
-  "maxBundleSize": "15kB",
+  "version": "0.8.0",
+  "main": "dist/f-http.umd.js",
+  "module": "dist/f-http.es.js",
```
Comment (DamianMullins) [inline packages/services/f-http/package.json:6]: 🚀

**Pull request merged**